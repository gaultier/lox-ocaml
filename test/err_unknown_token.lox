1 + $%.# ; // expect: 1:5:Unkown token: `$`
// expect: 1:6:Unkown token: `%`
// expect: 1:8:Unkown token: `#`

