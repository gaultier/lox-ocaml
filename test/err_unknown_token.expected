Unkown token: `$`
Unkown token: `%`
Unkown token: `#`
